### DISCUSSION TITLE
Live Sync not working on Plesk with Docker

### DISCUSSION THREAD

**Discussion by friedrichkeydel** [dani-garcia/vaultwarden#4570] [Q&A]
### Subject of the issue
I set up Vaultwarden using the integrated Docker interface of Plesk. The Container is running fine, but I don't know how to get Live Sync running!

### Deployment environment

* vaultwarden version: 1.30.2

* Install method: Docker image

* Clients used: web vault, desktop, iOS, Firefox Extension, Safari Extension

* Reverse proxy and version: Plesk Docker Rules

* Other relevant details:

### Steps to reproduce
Changes of password or username (e.g. inside organisation)

### Expected behaviour
The change should immediatly be reflected in all other clients. 

### Actual behaviour
Clients have to be syncronized manually to reflect the change.

### Troubleshooting data
My Plesk Docker configuration:
<img width="1015" alt="Bildschirmfoto 2024-05-17 um 22 37 47" src="https://github.com/dani-garcia/vaultwarden/assets/17936470/b94c2fe1-5bc1-4b69-ab64-a56a0d6971c2">

My Plesk Docker proxy rules:
<img width="1260" alt="Bildschirmfoto 2024-05-17 um 22 39 43" src="https://github.com/dani-garcia/vaultwarden/assets/17936470/dbad3994-21ea-4790-98f7-62182257eeb4">

**Comment by BlackDex** [comment]
I suggest to read https://github.com/dani-garcia/vaultwarden/discussions/4023

**Reply by friedrichkeydel** [reply]
I don't really see how this is related, as version 1.31 isn't released yet.

**Reply by BlackDex** [reply]
Al those feature are already in 1.30.5. best to configure it like that.

**Reply by friedrichkeydel** [reply]
I'm sorry, but I still can't get it to work, even with proxy configuration. I just don't know how to configure the proxy rules properly.
